opentech/static_src/src/app/src/redux/actions/submissions.js

```diff
@@ -85,7 +108,13 @@ export const loadCurrentRoundSubmissions = () => (dispatch, getState) => {
         return null
     }
 
-    return dispatch(fetchSubmissionsByRound(getCurrentRoundID(state)))
+    return dispatch(fetchSubmissionsByRound(getCurrentRoundID(state))).then(() => {
+        const state = getState()
+        const ids = getCurrentRoundSubmissionIDs(state)
+        if (!ids.includes(getCurrentSubmissionID(state))) {
+            return dispatch(setCurrentSubmission(null))
```

Comment:
I don't understand rationale for changing the current submission when fetching. Is that when user sets non-existing ID? Can we add a comment?

Reply:
This is the case where you try and look up a submission that isnt in the list anymore. i.e. its has changed status. 

I was considering adding a message at this point "The submission you are trying to view is now available in this status/round anymore"